spec/unit/resource/windows_audit_policy_spec.rb

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "spec_helper"
+
+describe Chef::Resource::WindowsAuditPolicy do
+  subcat_opts = ["Security State Change",
```

Comment:
Is this list the same as the one in the resource? If so, rather than copy-paste it the one in the resource should be made into a constant and referred to here. If they are not the same lists, I probably would expect them to be, so a comment explaining the difference would a be a good addition.